lib/src/runner/browser/browser.dart

```diff
@@ -120,6 +123,12 @@ abstract class Browser {
   Future close() {
     _closed = true;
 
+    // If we don't manually close the stream the test runner can hang.
+    // For example this happens with Chrome Headless.
```

Comment:
Did you file a `dart:io` bug for this? If so, refer to it here.

Reply:
Done.